app/assets/javascripts/application.js

```diff
@@ -11,7 +11,7 @@
 //= require topbar
 //= require two-pane
 //= require vendor
-//= require about
+//= require statuses
```

Comment:
status of what? just proxy status or hosts etc? (maybe a prefix to the name can help)